### PR TITLE
FIX: Workflow file for persistence performance

### PR DESCRIPTION
**Pull request by BenjaminBossan**
- wrong name
- trying to fix install

### CODE REVIEW

**Comment by BenjaminBossan** [comment]
@skops-dev/maintainers ready for review

**Comment by adrinjalali** [comment]
Could you please set this to trigger on PR so that we can test it here and see if the install actually fixes the issue?

**Comment by BenjaminBossan** [comment]
(don't merge yet)

@adrinjalali While the fundamental issue still exists, I found a "solution" by copying the script to the project root inside the GH action. WDYT?

**Comment by adrinjalali** [comment]
Since this is to "fix" import issues, would it work if we do an `os.chdir` or something inside the script instead, to run it from the root, before running the import statements? I'm not sure which one's better.

But I'm happy to merge and revert once https://github.com/actions/setup-python/issues/634 is resolved maybe?

File: .github/workflows/persistence-performance.yml
Comment (adrinjalali) [inline .github/workflows/persistence-performance.yml:30]: this probably can be `python3` or even `python`, right? You can probably revert these two lines to what it was before actually.

**Comment by BenjaminBossan** [comment]
@adrinjalali This works now, here is an output: https://github.com/skops-dev/skops/actions/runs/4531268169/jobs/7981214101?pr=326

I have removed the PR hook again, so we'll be 100% certain in 1 week :)

**Review by adrinjalali — APPROVED** [review]

**Pull request merged**
